fedmsg_meta_fedora_infrastructure/tests/releng.py

```diff
@@ -27,6 +27,187 @@
 
 
 class TestTwoWeekAtomicBegin(Base):
+    """ As one of the final steps in the `two week atomic
+    <https://fedoraproject.org/wiki/Changes/Two_Week_Atomic>`_ process, there
+    is a `release engineering script
+    <https://pagure.io/releng/blob/master/f/scripts/push-two-week-atomic.py>`_ that
+    runs to perform the release.
+
+    During the two week script run, it fetches AtomicHost artifacts detail
+    for given pungi compose ID using `fedfind
+    <https://pagure.io/fedora-qa/fedfind/>`_ . After getting image
+    artifacts and ostree commit information successfully for all supported
+    architectures, it sends this fedmsg indicating that Two Week release process has started.
+    """
+
+    expected_title = "releng.atomic.twoweek.begin"
+    expected_subti = "Release engineering scripts started evaluating a " + \
+        "new set of builds from Fedora 22 Atomic Host release"
+    expected_icon = "https://apps.fedoraproject.org/img/icons/atomic.png"
+    expected_secondary_icon = "https://apps.fedoraproject.org/img/icons/atomic.png"
+    expected_link = "https://download.fedoraproject.org/pub/alt/atomic/stable/"
+    expected_objects = set(['atomic_qcow2', 'atomic_raw'])
+
+    msg = {
+        u'timestamp': 1444939709.0,
+        u'topic': u'org.fedoraproject.prod.releng.atomic.twoweek.begin',
+        u'msg': {
+            u'aarch64': {
+                u'atomic_qcow2': {
```

Comment:
I forget, did we rule out putting ISO image data in the fedmsg? right here we only have qcow and raw image data

Reply:
No, I just added qcow2 and raw image for all arches to keep it a bit short. I am ok to add complete message which will include iso for all three arches and vagarant images detail for x86_64. Is that ok?

Reply:
ahh so the real fedmsg does have the ISO in it?

Reply:
yeah let's just make it match what the real fedmsg outputs. I think the fedmsg includes the ISO data, but if it doesn't we should probably add it.

Reply:
Existing fedmsg dosn't have ISO information. But we will have ISO information from next atomic twoweek release with recent change we did in push-two-week-atomic script. I will update this PR with exact content that we will have from our next atomic twoweek release fedmsg.

Reply:
yes, thanks